components/MastHead.js

```diff
@@ -110,44 +110,32 @@ export default function Home() {
             <div className="relative flex items-center justify-center mb-2 mx-2 md-12">
                 <div className="grid grid-cols-1 break-words">
                     <h2 id="start" className="text-2xl mt-10">Getting Started</h2>
-                    <ol className="list-decimal list-inside">
```

Comment:
I'm not sure I follow... are Git and Python not required on Windows?

Here is what I see:

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/774615/236991877-e5e00c83-8fc8-468d-a723-dc5e9395d14c.png">

Reply:
That is indeed the goal! I didn't realize they were included in the script.